registry/quilt_server/views.py

```diff
@@ -31,6 +31,7 @@
 from .models import (Access, Customer, Instance, Invitation, Log, Package,
                      S3Blob, Tag, UTF8_GENERAL_CI, Version)
 from .schemas import LOG_SCHEMA, PACKAGE_SCHEMA
+from .config import BAN_PUBLIC_USERS
```

Comment:
Sorry, one more thing... You should use `app.config` instead of importing it directly. See the code below.